File: Java/Logic-1/in1To10/In1To10.java
package in1To10;

public class In1To10 {
    //Se "outsideMode" for false, então retorne true se "n" estiver entre 1 e 10, inclusive, de maneira análoga, porém
    //se "outsideMode" for true, então retorne true se "n" for menor que 1 ou maior que 10.

    public boolean in1To10(int n, boolean outsideMode) {
        if (!outsideMode) return (n >= 1 && n <= 10);
        else return (n >= 10 || n <= 1);
    }
}

File: Python/Logic-1/in1to10.py
"""
Dado um número inteiro "n", retorne true caso ele esteja no intervalo entre 1..10,
inclusive, a menos que "outside_mode" seja true, nesse caso retorne true se "n"
for menor ou igual a 1 ou maior ou igual a 10.
"""


def in1to10(n, outside_mode):
    if outside_mode:
        return n >= 10 or n <= 1
    return 1 <= n <= 10


print(in1to10(1, True))
